### DISCUSSION TITLE
All in one

### DISCUSSION THREAD

**Discussion by shenweiyan** [shenweiyan/Digital-Garden#88] [2.1-乱弹]
写笔记，做表格的很多时候，总有一个 All in one 的魔咒在脑海挥之不去，以前用语雀就想当然 All in yuque，后来又想着 All in GitHub，但现实终究是现实。

没有十全十美的平台，语雀就没有邮箱、日历之类的功能；小记也没有浮墨笔记那么好用。

GitHub 虽好，但始终有一堵长城横亘在前面，时时刻刻提醒着你小心翼翼。

现在的飞书，各种功能都很齐全，但是不能实时查看文档的 Markdown，自己也没有能力去造这样一个轮子。有很多的时候，都有一个 All in feishu 的魔咒的诱惑着我，但又一次一次被它的木桶理论劝退。

知识的沉淀，对于承受的载体没有什么是可以从一而终一成不变的，很多东西虽然分散，但也印证了不要把所有的鸡蛋放进一个篮子。逐渐的，我用上了飞书国际版的邮箱，我用上了语雀（小记/知识库）来记录我的一些碎片想法和文字，最终还是选择 GitHub 来记录这一切。

不得不说，GitHub 真是个码字的好地方。

**Comment by R-957** [comment]
all in obsidian

**Reply by shenweiyan** [reply]
我想要一个在 web 端就能完成所有工作的工具，obsidian 很强大，但感觉不太适合我 >.<
